cmd/pipeline/execute.go

```diff
@@ -107,34 +106,5 @@ func executePipeline(cmd *cobra.Command, options ExecuteOptions) error {
 	if resp.StatusCode != http.StatusAccepted {
 		return fmt.Errorf("Encountered an error executing pipeline, status code: %d\n", resp.StatusCode)
 	}
-
-	executions := make([]interface{}, 0)
-	attempts := 0
-	for len(executions) == 0 && attempts < 5 {
-		executions, resp, err = gateClient.ExecutionsControllerApi.SearchForPipelineExecutionsByTriggerUsingGET(
-			gateClient.Context,
-			options.application,
-			map[string]interface{}{
-				"pipelineName": options.name,
-				"statuses":     "RUNNING",
-			})
-		attempts += 1
-		time.Sleep(time.Duration(attempts*attempts) * time.Second)
-	}
-	if err != nil {
-		return err
-	}
-	if resp.StatusCode < 200 || resp.StatusCode > 299 {
-		return fmt.Errorf("Encountered an error querying pipeline execution, status code: %d\n", resp.StatusCode)
-	}
-	if len(executions) == 0 {
-		return fmt.Errorf("Unable to start any executions, server response was: %v", resp)
-	}
-
-	if len(executions) > 1 {
-		return fmt.Errorf("Started more than one execution: %v", executions)
-	}
-
-	util.UI.JsonOutput(executions[0], util.UI.OutputFormat)
```

Comment:
Could we add something to indicate to the user that the request succeeded, similar to https://github.com/spinnaker/spin/blob/master/cmd/pipeline/save.go#L117

Reply:
Ah, yes, that makes sense: https://github.com/spinnaker/spin/pull/217/files#diff-217e0cd0e364fb48908d6a7e8e520cf6R110

(Thanks for the quick review!)